packages/components/src/styles/components/stepper/nav.scss

```
@@ -0,0 +1,326 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// STEPPER > NAV
+//
+
+@use "sass:map";
+@use "../../mixins/focus-ring" as *;
+@use "./step-indicator.scss" as *;
+
+.hds-stepper-nav__list,
+.hds-stepper-nav__step {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.hds-stepper-nav__list {
+  display: flex;
+}
+
+// STEPPER > NAV > STEP
+
+.hds-stepper-nav__step {
+  width: 100%;
+}
+
+.hds-stepper-nav__step__content {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 0;
+  background-color: transparent;
+  border: none;
+}
+
+.hds-stepper-nav__step__progress {
+  position: absolute;
+  top: -14px;
+  left: 0;
+  z-index: 0;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+
+  &::before,
+  &::after {
+    position: absolute;
+    top: 10px;
+    left: 0;
+    z-index: -1;
+    width: 100%;
+    height: 4px;
+    content: "";
+  }
+
+  &::before {
+    background-color: var(--token-color-palette-alpha-200);
```

We need to layer the alpha 200 with white behind it so its consistent regardless of where this is placed on top of:

```suggestion
    background: linear-gradient(0deg, var(--token-color-palette-alpha-200) 0%, var(--token-color-palette-alpha-200) 100%), --token-color-palette-neutral-100;
```